.github/workflows/performance-ci.yml

```diff
@@ -16,8 +16,6 @@ on:
       - 'v[0-9]+.[0-9]+'
 
 env:
-  BIGQUERY_DATASET: production_src_flow_tps_metrics
```

Comment:
This entire file can be removed, this yml now lives in dapperlabs/flow-go to make use of a private self-hosted runner. These changes are already implemented there.